.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -38,8 +35,10 @@ labels: bug
 
 <!--- Include as many relevant details about the environment you experienced the bug in -->
 
-<!-- ProTip: You can use `yarn backstage-cli info` command in your Backstage App for this section. -->
+- Browser Information: <!--- For example Google Chrome 97.0.4692.99 -->
+
+- Output of `yarn backstage-cli info`: <!--- Paste into the code block below  -->
+
+```text
 
-- NodeJS Version (v14):
-- Operating System and Version (e.g. Ubuntu 14.04):
-- Browser Information:
+```
```

Comment:
Very unsure about this bit, suggestions welcome :grin:

Goal here was to direct towards usage of `yarn backstage-cli info` more often

Reply:
I like it since it gives all the info we requested previously
```
$ /Users/jhaals/src/backstage/node_modules/.bin/backstage-cli info
OS:   Darwin 21.3.0 - darwin/arm64
node: v16.13.2
yarn: 1.22.1
cli:  0.13.2-next.0 (local)
Dependencies:
...
```
☝️ helpful!